.github/scripts/lint_test_ownership.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+'''
+Test ownership was introduced in https://github.com/pytorch/pytorch/issues/66232.
+
+This lint verifies that every Python test file (file that matches test_*.py or *_test.py in the test folder)
+has valid ownership information in a comment header. Valid means:
+  - The format of the header follows the pattern "# Owner(s): ["list", "of owner", "labels"]
+  - Each owner label actually exists in PyTorch
+  - Each owner label starts with "module: " or "oncall: " or is in ACCEPTABLE_OWNER_LABELS
+
+This file is expected to run in the root directory of pytorch/pytorch.
+'''
+
+import fnmatch
+import json
+import subprocess
+import sys
+from pathlib import Path
+from typing import List
+
+
+# Team/owner labels usually start with "module: " or "oncall: ", but the following are acceptable exceptions
+ACCEPTABLE_OWNER_LABELS = ["NNC", "high priority"]
+GLOB_EXCEPTIONS = [
+    "test/run_test.py"
+]
+
+PYTORCH_ROOT = Path(__file__).resolve().parent.parent.parent
+TEST_DIR = PYTORCH_ROOT / "test"
+CURRENT_FILE_NAME = Path(__file__).resolve().relative_to(PYTORCH_ROOT)
+
+
+def get_all_test_files() -> List[Path]:
+    test_files = list(TEST_DIR.glob("**/test_*.py"))
+    test_files.extend(list(TEST_DIR.glob("**/*_test.py")))
+    return [f for f in test_files if any([fnmatch.fnmatch(str(f), g) for g in GLOB_EXCEPTIONS])]
+
+
+def get_pytorch_labels() -> List[str]:
```

Comment:
Please decorate this function to make sure it is called only once
```suggestion
@functools.cache()
def get_pytorch_labels() -> List[str]:
```